test/plots/wealth-britain-proportion-plot.js

```diff
@@ -0,0 +1,69 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+
+export default async function() {
+  const wide = await d3.csv("data/wealth-britain.csv", d3.autoType);
+  const columns = wide.columns.slice(1);
+  const data = columns.flatMap(type => wide.map(d => ({age: d.age, type, value: d[type]})));
+  const stack = {
+    x: "type",
+    y: "value",
+    z: "age",
+    order: d3.sort(wide, d => d.shareA).map(d => d.age)
+  };
+  return Plot.plot({
+    marginLeft: 45,
+    marginRight: 55,
+    x: {
+      domain: columns,
+      axis: "top",
+      label: "",
+      tickFormat: d => d === "population" ? "Share of population" : "Share of national wealth",
```

Comment:
```suggestion
      tickFormat: d => `Share of ${d}`,
```

That way it’s a little less dependent on their being exactly two values in the domain?